b2b-sample/README.md

```diff
@@ -159,22 +159,12 @@ setup the config json file as mentioned above.
 
 ### Step 6: Setup the identity provider for the Guardio-Business-App
 
-* For this we will create an external identity provider in Asgardeo. After creating an identity provider in Asgardeo.
```

Comment:
`external identity provider in Asgardeo` -> `external identity provider in sub organization`

https://github.com/wso2/samples-is/pull/405/files#diff-34fe1b8cbee00497c7574755a2a49dc01937e2a04b08a16c2e7d3cfd24822197R162